torch/fx/graph.py

```diff
@@ -137,7 +137,7 @@ def node_copy(self, node: Node, arg_transform: Callable[[Node], Argument] = lamb
             # Placeholder names are user-visible, so they should be copied as-is without normalizing them.
             name = node.name
         else:
-            name = self._name(node.name)
+            name = self._name(node.target)
```

Comment:
what if my names are semantically meaningful (like some pass edited them) and I don't want to completely loose them?

I think it's better to actually track both versions in `_used_names`. E.g. you can add both with and without suffix to the dictionary (`{'add': 3, `add_1`: 0, `add_2`: 0, `add_3`: 0}`) - in the worst case it'd produce `add_1_1_!`.